rotkehlchen/db/dbhandler.py

```diff
@@ -1667,6 +1667,10 @@ def get_exchange_credentials(
                     f'the app with an older version. {e!s}',
                 )
                 continue
+
+            if location not in SUPPORTED_EXCHANGES:
```

Comment:
This is because we may add user credentials (like with monerium) that are not exchanges. It's already happening above with the rotki premium credentials (but they got no location, so it defaults to external).